util/init.js

```diff
@@ -4,6 +4,28 @@ const { Client } = require('pg');
 const opts = require('../config/appium-opts');
 const mobileServices = require('../config/mobile-services');
 
+const waitForDeviceReady = async done => {
+  const { deviceIsReady } = window.aerogear;
+
+  if (deviceIsReady) {
+    if (done) {
+      done();
+    }
+  } else {
+    await new Promise(resolve =>
+      document.addEventListener('deviceready', resolve, false)
```

Comment:
Any reasons why this part is also duplicated inside the index.js? https://github.com/jhellar/aerogear-integration-tests/blob/6559abd148b9e4c0560851951c489dea4113920c/fixtures/index.js#L40